psk/client-psk-resume.c

```diff
@@ -146,7 +121,19 @@ int main(int argc, char **argv){
     wolfSSL_set_fd(ssl, sockfd);
 
      /* takes inputting string and outputs it to the server */
-    SendReceive(ssl);
+    if (wolfSSL_write(ssl, sendline, sizeof(sendline)) != sizeof(sendline)) {
+		printf("Write Error to Server\n");
```

Comment:
Looks like there is an extra indent here.  Is this using spaces instead of tabs?